src/client/src/containers/DependencyInfo/index.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import styles from "./styles.module.css";
+import * as getSvg from "../../utils/getSvgUrl";
+import classnames from "classnames";
+import { injectIntl, defineMessages, InjectedIntl } from "react-intl";
+import { WIZARD_CONTENT_INTERNAL_NAMES } from "../../utils/constants";
+import { AppState } from "../../reducers";
+import { IDependenciesInstalled } from "../../reducers/dependencyInfoReducers";
+
+interface IDependencyInfoProps {
+  dependenciesStore: IDependenciesInstalled;
+  frameworkName: string;
+  intl: InjectedIntl;
+}
+
+type Props = IDependencyInfoProps;
+
+const messages = defineMessages({
+  installed: {
+    id: "dependencyChecker.installed",
+    defaultMessage: " detected!"
+  },
+  notInstalled: {
+    id: "dependencyChecker.notInstalled",
+    defaultMessage: " not detected. Click to install."
+  },
+  iconAltMessage: {
+    id: "dependencyChecker.iconAltMessage",
+    defaultMessage: "Icon for dependency checker"
+  }
+});
+
+interface IDependency {
+  commandName: string;
+  dependencyName: string;
+  downloadLink: string;
+  privacyStatementLink: string;
+}
+
+interface IDependencies {
+  [key: string]: IDependency;
+}
+
+const dependencies: IDependencies = {
+  NodeJS: {
+    commandName: "node", // command used on the extension side
+    dependencyName: "Node",
+    downloadLink: "https://nodejs.org/en/download/",
+    privacyStatementLink: "https://nodejs.org/en/about/privacy/"
+  },
+  Python: {
+    commandName: "python",
```

Comment:
Related to the previous comment, we may need to change more stuff (not sure where yet) given that the extension side checks for python using different commands (i.e. `python`, `python3` and `py -3`)

Reply:
Yea I agree, I really didn't want to touch the extension stuff as well in this pr

Basically `commandName` is the attribute for the payload.dependency that we pass back and forth between client and extension, on the extension side you take this `commandName` (payload.dependency), check it, then run it directly.

Reply:
Pretty sure command name is just a dictionary key. It isnt actually executed. We group all of the python commands as python and all node as node. It doesn't seem to be sending any requests to the extension. I assume this means the data already exists. Maybe a better name would help you? dependencyStoreKey or something?

Reply:
Unless this code is meant to send requests but doesn't yet in which case you'd be right.. don't see any dispatching here though. Think this is just meant to grab the data we want and map it to the right framework using the dependencyKey (commandName)

Reply:
I needed a separate attribute `displayName` that's capitalized to display it

Reply:
sorry for the confusion, this `commandName` isn't the one actually executed, but it is same as the one you execute from the payload I send from the client